clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/view/GoogleInstanceProvider.groovy

```diff
@@ -120,9 +121,9 @@ class GoogleInstanceProvider implements InstanceProvider<GoogleInstance.View> {
       }
     }
 
-    def serverGroupKey = cacheData.relationships[SERVER_GROUPS.ns]?.first()
-    if (serverGroupKey) {
-      instance.serverGroup = Keys.parse(serverGroupKey).serverGroup
+    def serverGroup = GCEUtil.getLocalName(cacheData.attributes.metadata?.items?.find { it.key == "created-by" }?.value)
```

Comment:
Does this ever point to a user/service-account?

Reply:
Not as far as I can tell. I have added a TODO to replace this with whatever unique key we end up adding to enable easier grouping via stackdriver (this is happening shortly).